src/main/java/racingcar/domain/car/RacingCarFactory.java

```diff
@@ -0,0 +1,10 @@
+package racingcar.domain.car;
+
+import racingcar.domain.car.impl.RacingCarImpl;
+
+public class RacingCarFactory {
+
+    public static RacingCarImpl createRacingCar(String carName) {
```

Comment:
향후에 Java에 있는 다형성을 사용하기 위해서 아래와 같이 추상클래스를 반환형으로 지정하는 것은 어떤가요 ~ ?

```suggestion
    public static RacingCar createRacingCar(String carName) {
```